controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -221,10 +221,11 @@ http {
         {{ $path := buildLocation $location }}
         {{ $authPath := buildAuthLocation $location }}
 
-        {{ if not (empty $location.CertificateAuth.CertFileName) }}
+        {{ if not (empty $location.CertificateAuth.CAFileName) }}
         # PEM sha: {{ $location.CertificateAuth.PemSHA }}
         ssl_client_certificate              {{ $location.CertificateAuth.CAFileName }};
         ssl_verify_client on;
+        ssl_verify_depth 10;
```

Comment:
@euank You're right, i've been pretty lazy here. I'll add another annotation instead of hardcoding this, so the user can choose what it's verify depth.

Just FYI, here we use ssl_verify_depth 3 as he have the structure root -> intermediate -> final -> client :)